agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2313,6 +2319,66 @@ bool backhaul_manager::handle_ap_capability_query(ieee1905_1::CmduMessageRx &cmd
     return send_cmdu_to_bus(cmdu_tx, src_mac, bridge_info.mac);
 }
 
+bool backhaul_manager::handle_ap_metrics_query(ieee1905_1::CmduMessageRx &cmdu_rx,
```

Comment:
Unless I'm missing something crucial - this function and the slave_thread function with the same name (6f9a662#diff-f3dcf05057f5ecc09b7bc39daa3fe7f2R4528) are doing the same:

- verification on the rx
- build tx from the data inthe rx.

Maybe it would be better to separate this functionality into one function that both threads would use?